recipes/substring/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "substring" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/substring-{{ version }}.tar.gz
+  sha256: 4eae4420043329b7a24f6151c1c372d7319979c4dd5c29e3ed73d4c272de0c59
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I could not find any compiled extensions that require a compiler here. Also, it cannot be noarch if this is really needed.

```suggestion
```